Capstone1

CesarHerr opened the pull request:
- Personalize the content of your page. Choose a topic that is different than the one in the original design. ✔️ 

- The Home page. ✔️ 
- The About page. ✔️ 

Each of these pages should have versions for 2 different screen sizes:

- Mobile: up to 768px wide.✔️ 
- Desktop: 768px or wider.✔️ 

Interactions

- The home page should have a link in the menu to the about page. ✔️ 
- The logo in the header links to the home page. ✔️ 

Mobile menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page. ✔️ 
- The mobile menu should have a close (X) button that closes the menu. ✔️ 

Dynamic page

- The section "Featured speakers" should be created dynamically in JavaScript.✔️ 
- Use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically. ✔️ 
- Give appropriate credit to the author. ✔️ 
- Record a video presenting the features of the project you built. ✔️ 
   https://www.loom.com/share/f32174e0840445a19c3f60cb60269dda

CesarHerr committed:
Add created new branch, index file, and installed linters

CesarHerr committed:
Update Index file

Add created html structure

CesarHerr committed:
Created File

Add file scss, applied reset to  styles

CesarHerr committed:
Update scss file

Add mixins to the scss file

CesarHerr committed:
update scss and css file

Add styles for main mobile version

CesarHerr committed:
Created file

Add created about.html file and its structure

CesarHerr committed:
Update about file

Add styles for about mobile version

CesarHerr committed:
Updated files

Add created hamb button on main and about pages

CesarHerr committed:
Update Scss files

Add styles for desktop vertion on main page

CesarHerr committed:
Update scss file

Add styles for about desktop version

CesarHerr committed:
Update js files

Add created features section dynamically, and button 'see more' for the movil version

CesarHerr committed:
Updated files

Add changes to html information, and changes styles on desktop versions

CesarHerr committed:
Update js and css files

Add fix more button and created less button and styles

CesarHerr committed:
Created Readme file

Add Created Readme file

CesarHerr committed:
update readme file

Add fixed Readme file information

CesarHerr committed:
Update css and js files

Add fixed isues whit linters

CesarHerr committed:
Update Readme file

Add video presentation to the readme file

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,114 @@
+            </ul>                
+            <button class="hamburger-open"><img src="./img/Hamburger_icon.png" alt="hamburger-icon"></button>                
+        </nav> 
+    </header>
```
Nice work so far. Kindly check that your design has the same margin, alignment and positioning as the actual design. 
The margin to the left side of both the top and bottom bars is smaller. Kindly add a right margin to match the design and make sure they are aligned as in the actual design. :wink: 



### Your design 

![Screenshot from 2023-04-20 09-22-06](https://user-images.githubusercontent.com/65233860/233322369-5b7a2c6f-e280-47be-88de-06a45fe9d4ae.png)




### Actual design 


![Screenshot from 2023-04-20 09-21-55](https://user-images.githubusercontent.com/65233860/233322391-78a6daca-41fd-447a-8c0d-96a86fc709fd.png)

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,114 @@
+                <li><a href="#">Join</a></li>
+                <li><a href="#partners">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#">CC Campaign</a></li>                                    
```
In the actual design, this `CC Campaign` should have a font and border of #EC5242 (color). Kindly update the design to match it.

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,114 @@
+    <footer class="footer">
+        <img src="./img/Unlimited2.svg" alt="unlimited_logo">
+        <p>2023. Creative CesCon. <br> Some rights Reserved.</p>
+    </footer>    
```
The mobile version of the index page does not have the partners and the footer since it ends at the more button as shown below :point_down: Kindly update your project to match the design

### Your design 

![Screenshot from 2023-04-20 09-35-51](https://user-images.githubusercontent.com/65233860/233324640-8a4170f2-6c3b-4c8d-91fe-d1584b715916.png)


### Actual design 

![Screenshot from 2023-04-20 09-36-01](https://user-images.githubusercontent.com/65233860/233324668-d421efc5-454c-45c5-8518-872fd4829358.png)

jerryowusu commented on about.html:
```diff
@@ -0,0 +1,87 @@
+    <footer class="footer hidden">
+        <img src="./img/Unlimited2.svg" alt="unlimited_logo">
+        <p>2023. Creative César Herrera. <br> Some rights Reserved.</p>
+    </footer>
```
Kindly check and make the following changes to match the about page design. 

- [ ] The desktop version of the about page does not have any partners section as shown in the image below. Please consider removing it. 
- [ ] The footer of the desktop version of the about page has a dark background as shown below :point_down: . Kindly update the background color to match the design. 

### Your design 

![Screenshot from 2023-04-20 09-38-43](https://user-images.githubusercontent.com/65233860/233326622-b5132d28-8b2e-42d2-9d81-335b5f8c1eba.png)



### Actual design 

![Screenshot from 2023-04-20 09-38-15](https://user-images.githubusercontent.com/65233860/233326668-cb11b336-525d-4475-be7b-72842ced8d9f.png)

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,114 @@
+    </header>
+
+    <main>
+        <section class="headline">            
+                <h2>"Hello hikers of the world!"</h2>
+                <h1>HIKING AND ADVENTURE SPORTS<br> SEASON 2023</h1>            
+                <p class="headline__p">Hiking and trekking · Ski and snowboard · Mountainbike · Horseback riding · Mountaineering and climbing · Surf · Sport fishing · Kayak <br> Are part of the adventure that awaits you this new season</p>
+                <h3>2023.04.18(thu) ~ 21(FRI)</h3>
+                <p>@Pucón, región de la Araucanía, Chile</p>            
+        </section>
```
The font-size of this section looks bigger than the actual design. Kindly consider decreasing the font-size and also increasing the width especially the width of the paragraph to match the design 

### Your design 

![Screenshot from 2023-04-20 09-50-02](https://user-images.githubusercontent.com/65233860/233330717-dbe7ab34-b64f-4d39-8ac4-5f2f4be6a642.png)


### Actual design 


![Screenshot from 2023-04-20 09-50-13](https://user-images.githubusercontent.com/65233860/233330767-7165b1b0-e16a-4022-b071-5d1c056c1704.png)

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,114 @@
+        </section>
+
+        <section class="program" id="program">            
+                <h2>Main Program</h2>
+                <hr>
+                <ul class="program__ul">
+                    <li class="program__ul--bike">
+                        <img src="./img/biking.svg" alt="biking">
+                        <h3>Mountain bike</h3>
+                        <p>The full suspension bicycle represents the most important advance in cycling in the 20th century; since it provides cyclists with the possibility of reaching places that in another time were considered inaccessible to said vehicle.</p>
+                    </li>
+                    <li>
+                        <img src="./img/ski.svg" alt="skiing">
+                        <h3>Skiing</h3>
+                        <p>Skiing is a mountain sport that consists of sliding through the snow, by means of two boards attached to the sole of the skier's boots by means of mechanical-robotic fixings, with multiple buttons with various functions.</p>
+                    </li>
+                    <li>
+                        <img src="./img/volcano.svg" alt="volcano">
+                        <h3>Villarrica Volcano</h3>
+                        <p>Visible from miles away, though if you spot a cheeky spurt, you may want to wait a few days until you try to conquer this monstrous, active volcano! The hike is around five hours, and is possible without prior training.</p>
+                    </li>
+                    <li>
+                        <img src="./img/water.svg" alt="water">
+                        <h3>Water sports</h3>
+                        <p>White-water rafting, Canyoning, kayaking, hydrospeed, are some of the water sports that you can enjoy.                      </p>
+                    </li>
+                    <li>
+                        <img src="./img/hiking.svg" alt="hiking">
+                        <h3>Hiking</h3>
+                        <p>Nestled on the shores of Lake Villarrica in Chile, Pucón is heaven for outdoor lovers. It is home to hot springs, active volcanoes, and several national parks. In this article, I will share my hiking and trekking experience in the Lake region of Chile.</p>
```
At certain screen widths of the desktop version, the main program becomes two rows with 3 on top and two below making it different from the actual design. Kindly make sure all the cards stay in one row just as in the original version at all screen sizes of the desktop. 

### Your design 

![Screenshot from 2023-04-20 10-04-36](https://user-images.githubusercontent.com/65233860/233333434-546ed3e6-fc13-433e-b87b-7c8aa7172bd0.png)


### Actual design 

![Screenshot from 2023-04-20 09-57-27](https://user-images.githubusercontent.com/65233860/233333460-246ceeff-cdf8-4139-833f-b48f95d212b4.png)

jerryowusu commented on scss/styles.css:
```diff
@@ -0,0 +1,686 @@
+  margin-bottom: 1em;
+}
+
+.speakers img {
+  float: left;
+  margin: 10px;
+  display: inline;
+  width: 25%;
+  height: 8em;
+  position: relative;
+  -o-object-fit: fill;
+  object-fit: fill;
+  border-radius: 0 2rem 0 2rem;
+  border: solid 2px #312827;
+  box-shadow: 15px 15px 15px rgba(0, 0, 0, 0.15);
+}
```
In the original design, there is a checquered background behind the image of the speaker which is missing in your design as shown below :point_down: . Kindly update accordingly. 

### Your design

![Screenshot from 2023-04-20 10-15-58](https://user-images.githubusercontent.com/65233860/233336312-8f033069-ff88-428f-9873-6e4b97e46d6a.png)


### Actual design 

![Screenshot from 2023-04-20 10-16-08](https://user-images.githubusercontent.com/65233860/233336345-58c34148-cd4c-4475-b365-98b43fc1147e.png)

jerryowusu commented on scss/styles.css:
```diff
@@ -0,0 +1,686 @@
+}
+
+.header__links {
+  position: fixed;
+  width: 100%;
+  background-color: rgba(211, 211, 211, 0.8470588235);
+  z-index: 10;
+}
+
+@media screen and (min-width: 768px) {
+  .header__links {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: flex-end;
+    align-items: center;
+    position: relative;
+    background-color: rgba(255, 255, 255, 0.9411764706);
+  }
+}
+
+.header__links .hamburger-open {
+  width: 60px;
+  background: none;
+}
+
+@media screen and (min-width: 768px) {
+  .header__links .hamburger-open {
+    display: none;
+  }
+}
+
+.header__links .show-icon {
+  display: none;
+}
+
+.header__links--logo {
+  display: none;
+  position: absolute;
+  z-index: 20;
+  left: 50%;
```
I am really impressed with your design skills so far. As stated in the project requirements shown below, you are allowed to personalized the content/text,  however, the pages should use the exact same colors, for text, backgrounds, borders, etc, as the original design. All page elements should also  have similar spacing, borders, shadows, and paddings as the orignal design. 
Kindly check and make sure your design follows this rule. I can see a lot of stylng differences in your project which does not match the original design. A few of them are illustrated below and must be updated. 

- [ ] You have used the border-radius style shown below throughout your project, but it is not seen anywhere in the design. 


### Your design 

![Screenshot from 2023-04-20 10-22-33](https://user-images.githubusercontent.com/65233860/233342468-3fc830af-6424-4c74-9f13-63a57c1d17a7.png)


### Actual design with no border radius

![Screenshot from 2023-04-20 10-22-19](https://user-images.githubusercontent.com/65233860/233342529-9622016f-09b9-41fd-ab69-f79ebb28bfb4.png)

jerryowusu reviewed (changes requested):
Hi @CesarHerr 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:


![wow so close](https://user-images.githubusercontent.com/65233860/233343596-4a01e49e-3fd7-4c96-b8f7-b306c3752f0e.gif)


## Highlights
Neat and well organized code :+1: 
Great job so far :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

CesarHerr committed:
Update scss file

Add fixed margins header, fixed border cc Campaign button, fixed button 'more'

CesarHerr committed:
update css, html and js files

Add fixed partners and footer section on about, fonts size and widht on main page, design on 'main program' section, background on image of the speakers

CesarHerr committed:
Update html, css and scss files

Add changed spacings, borders, shadows , paddings and colors to match whit the original design

nahnahdev commented on about.html:
```diff
@@ -0,0 +1,91 @@
+<body>
+    <header class="header">
+        <div class="header__social">
+            <ul class="header__social--ul">
+                <li>
+                    <img class="facebook" src="./img/facebook.svg" alt="facebook">
+                    <img src="./img/twitterWhite.svg" alt="Twitter">
+                </li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
```
- [ OPTIONAL ] Great job so far @CesarHerr Your project looks great. I will only suggest that just as you adjusted the styling of the top bar to be vertically aligned with the nav bar as the previous reviewer mentioned on the Home page, I will suggest you do the same on the About Page as well :wink: 
- Your Design Now
![image](https://user-images.githubusercontent.com/102699958/233485108-ab7529d3-9ead-41f4-ac9c-ef61f46e0c4e.png)
 - Your Home Page
![image](https://user-images.githubusercontent.com/102699958/233485399-9edb1a4a-c156-4977-9a70-fffd7bf08f05.png)

CesarHerr replied:
@NahnahAJ    Thank's for the suggestion and for the review 🙇 , I'll do this ☝️ .

nahnahdev reviewed (approved):
## Project Status: APPROVED :green_circle: 
Hi @CesarHerr , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media3.giphy.com/media/3oz9ZE2Oo9zRC/giphy.gif"/>

## Highlights
- Amazing Design :fire: 
- All requested changes implemented :fire: 
- No linter errors :fire: 
- Project completed :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
